src/ucp/core/ucp_ep.h

```diff
@@ -176,4 +175,20 @@ ucp_md_map_t ucp_ep_config_get_rma_md_map(const ucp_ep_config_key_t *key,
 ucp_md_map_t ucp_ep_config_get_amo_md_map(const ucp_ep_config_key_t *key,
                                           ucp_lane_index_t lane);
 
+static UCS_F_ALWAYS_INLINE
+size_t ucp_ep_config_get_zcopy_auto_thresh(double iovcnt,
+                                           const uct_linear_growth_t *reg_cost,
+                                           double bcopy_bw, double bandwidth)
```

Comment:
I did this to avoid explicit conversion from `size_t` to `double`. Originally, `bcopy_bw` has `size_t` type but we need it as `double` in calculations.
I will change it as you like shortly.